scib/metrics/metrics.py

```diff
@@ -147,36 +187,72 @@ def metrics(
         adata_int,
         batch_key,
         label_key,
-        hvg_score_=False,
+        embed='X_pca',
         cluster_key='cluster',
         cluster_nmi=None,
         ari_=False,
         nmi_=False,
```

Comment:
Why do these arguments have underscores?

Reply:
I think it was initially to make sure that they don't get confused with the functions themselves, but I'm not sure if that's actually a problem. Might consider changing these parameters, which would also require changes in scib-pipeline